components/announcement_bar/index.ts

```diff
@@ -11,6 +11,8 @@ import {getDisplayableErrors} from 'mattermost-redux/selectors/errors';
 import {dismissError} from 'mattermost-redux/actions/errors';
 import {getStandardAnalytics} from 'mattermost-redux/actions/admin';
 import {GenericAction} from 'mattermost-redux/types/actions';
+import {getCloudSubscription, getCloudCustomer, getSubscriptionStats} from 'mattermost-redux/actions/cloud';
```

Comment:
I like the approach of moving all the calls and `withGet` wrapping one level up from the child bars to the parent controller. As part of that, I think the HOC should start making use of getCloudCustomer similar to how `payment_announcement_bar` componentDidMount does it, and remove the current payment_announcement_bar call.

Reply:
I see what you're saying - I think the reason this is like this is that the `payment_announcement_bar` is the only bar that actually utilizes the customer data - everything else is subscription only. I think this is a bit out of scope for this small change - you ok if I open a ticket for it and we can look into it later?

Reply:
oh, sure.

Reply:
Here's that ticket: https://mattermost.atlassian.net/browse/MM-42602